depthai_nodes/ml/parsers/utils/yolo.py

```diff
@@ -276,7 +358,36 @@ def sigmoid(x: np.ndarray) -> np.ndarray:
     return 1 / (1 + np.exp(-x))
 
 
-def process_single_mask(protos: np.ndarray, mask_coeff: np.ndarray, mask_conf: float):
+def crop_mask(mask: np.ndarray, bbox: np.ndarray) -> np.ndarray:
```

Comment:
Should this be moved to some other location in the utils? I don't think this is YOLO specific